src/parsers/containers/isobmff/utils.ts

```diff
@@ -45,6 +49,14 @@ export interface IISOBMFFPSSHInfo {
   privateData : Uint8Array;
 }
 
+export interface IEMSG { schemeId: string;
```

Comment:
As we're talking about an exposed API here, I think we should use the exact same terms than in the standard.

Isn't this one more `schemeIdUri`?